src/app/rpmostree-builtin-start-daemon.c

```diff
@@ -50,40 +58,43 @@ static GOptionEntry opt_entries[] =
 static RpmostreedDaemon *rpm_ostree_daemon = NULL;
 
 static void
-start_daemon (GDBusConnection *connection)
+state_transition (AppState state)
```

Comment:
Double logging tends to bug me; systemd itself already logs e.g.:

```
Mar 06 14:22:16 localhost.localdomain systemd[1]: Stopping OSTree+RPM package management daemon...
Mar 06 14:22:16 localhost.localdomain systemd[1]: Stopped OSTree+RPM package management daemon.
```

(And not to mention auditd *also* logs service stops)

I'm not sure we'd be adding much value on top of those.  That said, how about `sd_notify ("STATUS")` ?  See ⬇️